txircd/modules/extra/services/account_nick_protect.py

```diff
@@ -58,8 +62,12 @@ def resolveNickProtection(self, user, nick):
 			return
 		if self.userSignedIntoNickAccount(user):
 			return
-		user.changeNick(user.uuid)
-		recoverSeconds = self.ircd.config.get("account_nick_recover_seconds", 10)
+		newNick = self.ircd.config["account_nick_default_prefix"] + user.uuid
+		if newNick in self.ircd.userNicks:
+			user.changeNick(user.uuid)
+		else:
+			user.changeNick(newNick)
+	recoverSeconds = self.ircd.config.get("account_nick_recover_seconds", 10)
```

Comment:
It looks like you unindented setting `recoverSeconds` here, which will break setting the recovery timeout.

Reply:
Aah, oops. Got shift-happy